docs/2_3_fmu_distribution.adoc

```diff
@@ -55,12 +55,12 @@ binaries                      // directory containing the binaries (optional)
    // folder are recommended to be unpacked at the same location as the binary
    // < modelIdentifier >.* is unpacked. If not it is likely that, if the FMU
    // has dependencies on those items, it will not be able to find them.
-resources                     // resources needed by the FMU
+resources                     // resources used by the FMU (optional)
    // data in FMU specific files which will be read during initialization;
    // also more folders can be added under resources (tool/model specific).
    // In order for the FMU to access these resource files, the resource directory
-   // must be available in unzipped form and the absolute path to this directory
-   // must be reported via argument "resourceLocation" of fmi3InstantiateXXX.
+   // shall be available in unzipped form and the absolute path to this directory
+   // can be reported via argument "resourceLocation" of fmi3InstantiateXXX.
```

Comment:
```suggestion
   // should be reported via argument "resourceLocation" of fmi3InstantiateXXX.
```

Reply:
Can seems to lenient, since this implies an implementation is completely free to chose whether to supply it, whereas we probably want to state that it really should, unless the stated reasons in the non-normative sections apply...